pkg/dyninst/integration_test.go

```diff
@@ -53,13 +68,190 @@ func TestDyninst(t *testing.T) {
 			if cfg.GOARCH != runtime.GOARCH {
 				t.Skipf("cross-execution is not supported, running on %s", runtime.GOARCH)
 			}
-			bin := testprogs.GetBinary(t, "simple", cfg)
-			testDyninst(t, bin)
+			bin := testprogs.GetBinary(t, "sample", cfg)
+			t.Logf("loading binary")
```

Comment:
This log line is a bit odd given nothing that follows it involves loading a binary.